src/WatchedSetting.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export interface WatchedSetting {
+    key: string;
+    label?: string;

```

Comment:
JS SDK:
In client.listConfigurationSettings(options) , it accepts labelFilter as `\0` for "no label".
![image](https://github.com/Azure/AppConfiguration-JavaScriptProvider/assets/2351748/c18529ee-9c38-4148-bf06-10ed607cd3d0)

while in the response, "no label" represented by `null`:
![image](https://github.com/Azure/AppConfiguration-JavaScriptProvider/assets/2351748/61a78006-7906-496f-bc9d-fed2fd51841c)

When not specified, e.g. `{key: "some-key"}`, then the label is `undefined`. 

Here we must consider how to deal with these values and well-document them:
- `\0`
- `null`
- `undefined`

Reply:
![image](https://github.com/Azure/AppConfiguration-JavaScriptProvider/assets/2351748/11bd5220-5d5b-466b-a7fa-7dcffa73a86f)

Here by omitting it, it means "no label" instead of "any" as above in LabelFilter... Ideally I would expect something to be done in upstream JS SDK, to eliminate the inconsistency...

Reply:
created a ticket in upstream to track: https://github.com/Azure/azure-sdk-for-js/issues/27607

Reply:
azure-app-configuration v1.5.0 released containing the fix. we should update.

Reply:
update with PR https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/27

Reply:
fixed by upstream sdk.